playground/conversions.ts

```diff
@@ -1,13 +1,13 @@
 import * as THREE from 'three'
 import { Units, InputTypes } from './types'
 import { rotations } from './rotations'
-// import { OrientationVector } from '../src/main'
+import { OrientationVector } from '../src/main'
```

Comment:
This file demonstrates usage, and will be a playground allowing users to convert between rotation types. I'll request setting up a URL once this PR is merged.